src/validations/user.validation.js

```diff
@@ -179,6 +180,58 @@ class UserValidation {
     }
     next();
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @param {next} next
+   * @returns {validation} This function validate view comments route
+   * @memberof UserValidation
+   */
+  static async validateViewComment(req, res, next) {
+    const page = parseInt(req.query.page, 10);
+    const limit = parseInt(req.query.limit, 10);
+
+    const schema = Joi.object({
+      tripId: Joi.number().greater(0).required(),
+      page: Joi.number().greater(0).required()
+        .messages({
+          'number.greater': 'Page must be greater than 0',
+          'object.unknown': `${page} is not allowed`,
+          'any.required': 'Page is required',
+          'number.base': 'Page must be a number'
+        }),
+      limit: Joi.number().greater(0).required()
+        .messages({
+          'number.greater': 'Limit must be greater than 0',
+          'object.unknown': `${limit} is not allowed`,
+          'any.required': 'Limit is required',
+          'number.base': 'Limit must be a number'
+        })
+    }).options({ abortEarly: false });
+
+    const { error } = schema.validate({ ...req.params, ...req.query });
+    if (error) {
+      const { details } = error;
+      const errors = details.map(({ message }) => message.replace(/[^a-zA-Z0-9 .-]/g, ''));
+      ResponseService.setError(400, errors);
+      return ResponseService.send(res);
+    }
+
+    const comment = await CommentService.findCommentByProperty({
```

Comment:
Put this in its own validation file. Also, I'm curious to know why the `validateViewComment` validator is put inside `user.validation.js`  file